internal/pkg/rbldap/ldap.go

```diff
@@ -0,0 +1,40 @@
+package rbldap
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/redbrick/rbldap/pkg/rbuser"
+	"github.com/urfave/cli"
+	ldap "gopkg.in/ldap.v2"
+)
+
+func newRBLdap(ctx *cli.Context) (*ldap.Conn, error) {
```

Comment:
Would be better to abstract it away so it just takes the path to LDAP, not cli context. But I'm fine with this being in another PR.

Reply:
possibly. the initial idea of this lib was to just be to parse the `context` for `rbuser` but i dont really mind